deployment/human-connection/maintenance/README.md

```diff
@@ -10,9 +10,10 @@ bring the database into maintenance mode for manual database migrations.
 ## Deploy the service
 
 We prepared sample configuration, so you can simply run:
+
 ```sh
 # in folder deployment/
-kubectl apply -f human-connection/maintenance
+$ kubectl apply -f human-connection/maintenance
```

Comment:
Is this correct? I would expect `ocelotsocialnetwork/maintenance`

Reply:
Yeah that’s right @Mogge ! 👍🏼
`ocelotsocialnetwork/develop-maintenance`

I’ve done it now local …